jax/_src/lax/windowed_reductions.py

```diff
@@ -301,19 +384,113 @@ def _generic_reduce_window_batch_rule(
       *(operands + init_values), jaxpr=jaxpr, consts=consts,
       window_dimensions=window_dimensions, window_strides=window_strides,
       padding=padding, base_dilation=base_dilation,
-      window_dilation=window_dilation)
+      window_dilation=window_dilation, is_jvp=is_jvp)
   return outs, (0,) * num_operands
 
 
 reduce_window_p = core.Primitive('reduce_window')
+
+
+def reduce_window_jvp(
+    primals,
+    tangents,
+    window_dimensions,
+    window_strides,
+    padding,
+    base_dilation,
+    window_dilation,
+    jaxpr,
+    consts,
+    is_jvp,
+):
+  if is_jvp:
+    raise NotImplementedError("Re-entrant JVP NYI")
```

Comment:
Is this trying to say that higher-order AD isn't supported? If so, maybe writing something closer to that would be clearer.